lib/messager.rb

```diff
@@ -0,0 +1,21 @@
+class Messager
+
+  require 'twilio-ruby'
+  require 'dotenv'
+
+  def initialize
+    @credentials = Dotenv.load
+  end
+
+  def send_sms(message)
+
+    @client = Twilio::REST::Client.new @credentials["ACCOUNT_ID"], @credentials["AUTH_TOKEN"]
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)